tests/printer/expr/expected/let.res.txt

```diff
@@ -47,11 +47,7 @@ let f = x => {
     ->Option.flatMap(Js.Json.decodeString)
     ->Option.map(Js.Date.fromString)
 
-  let b =
-    x
-    ->Js.Dict.get("wm-property")
-    ->Option.flatMap(Js.Json.decodeString)
-    ->Option.flatMap(x =>
+  let b = x->Js.Dict.get("wm-property")->Option.flatMap(Js.Json.decodeString)->Option.flatMap(x =>
```

Comment:
AFAIKT this would be a big regression. The community loves tidy pipe chains on multiple lines after a certain line width has been reached.

But I guess this causes most of the perf issues?

Reply:
There's no causality.
It's an unintended effect of how the exponential explosion is avoided.